test/noted/notes_test.exs

```diff
@@ -75,11 +79,56 @@ defmodule Noted.NotesTest do
       {:ok, tag} =
         attrs
         |> Enum.into(@valid_attrs)
-        |> Notes.create_tag()
+        |> Notes.create_tag!()
 
       tag
     end
 
+    def get_notes_tags(note, tag) do
+      Repo.one!(from n in NotesTags, where: n.note_id == ^note.id and n.tag_id == ^tag.id)
+    end
+
+    test "add_tag/3 add the tag to a note" do
```

Comment:
With setup block, add , ctx for accessing context data from setup.

ctx.note, ctx.user, etc.